### PR TITLE
Add project links on setup.py for PyPI

### PR DESCRIPTION
**Pull request by Kludex**
Idea "copied" from [django-rest-framework](https://github.com/encode/django-rest-framework/blob/98e56e0327596db352b35fa3b3dc8355dc9bd030/setup.py#L109-L112) thanks to @graingert points. :)

### CODE REVIEW

**Review by euri10 — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -95,4 +95,8 @@ def get_packages(package):
     [console_scripts]
     uvicorn=uvicorn.main:main
     """,
+    project_urls={
+        "Funding": "https://github.com/sponsors/encode",
+        "Source": "https://github.com/encode/uvicorn",
```
Comment (graingert) [inline setup.py:100]: changelog?

**Review by graingert — CHANGES_REQUESTED** [review]
Changelog?

**Comment by euri10** [comment]
> Changelog?

agreed, that goes with my comment here https://github.com/encode/uvicorn/pull/1134

**Comment by euri10** [comment]
also my brain farts seeing that many emojis in PR titles, but this is unrelated

**Review by graingert — APPROVED** [review]

**Comment by Kludex** [comment]
@euri10 I can stop using gitmoji if it bothers you. I don't mind. Just let me know 😗

**Comment by graingert** [comment]
I'm not sure what ♿ means in terms of setup.py change. This is because it changes wheel metadata?

**Comment by Kludex** [comment]
Well, maybe it was not the best choice... That one means "improve accessibility". But it could be the 📦.

Ref.: https://gitmoji.dev/

**Comment by euri10** [comment]
> @euri10 I can stop using gitmoji if it bothers you. I don't mind. Just let me know

you do as you want @kludex I just find all those emoji eveywhere for everything quite unreadable, unnecessary.
good read: https://www.tandfonline.com/doi/abs/10.1080/014492999119129

**Comment by Kludex** [comment]
I'm going to read it later. Thanks!

But as I said, I don't mind. So if it's improve your experience, I'll stop using it.

**Pull request merged**